components/TProfile.vue

```diff
@@ -117,15 +127,25 @@
             {{ profile.name }}
           </div>
           <div class="ml-2 text-sm text-gray-700">
-            <span v-if="profile.height">• {{ profile.height }}cm</span>
-            <span v-if="profile.weight">• {{ profile.weight }}kg</span>
+            <span v-if="profile.height"
+              >•
+              {{
+                $t('myprofile.profile.height', { height: profile.height })
```

Comment:
```suggestion
                $t('profile.height', { height: profile.height })
```

Reply:
there was already a profile.height key

Reply:
profile.height.cm ? since profile.height is already being used

Reply:
ok, leave as is